rl4co/models/rl/reinforce/baselines.py

```diff
@@ -53,8 +53,8 @@ def eval(self, td, reward, env=None):
 class SharedBaseline(REINFORCEBaseline):
     """Shared baseline: return mean of reward as baseline"""
 
-    def eval(self, td, reward, env=None, on_dim=1):  # e.g. [batch, pomo, ...]
-        return reward.mean(dim=on_dim, keepdims=True), 0
+    def eval(self, td, reward, env=None):  # e.g. [batch, pomo, ...]
+        return reward.mean(), 0
```

Comment:
Isn't the code after the change wrong?

![image](https://github.com/kaist-silab/rl4co/assets/40595684/a5dbd24d-0f19-4133-8424-2496e0e7667f)

Reply:
Yes, that's what I think. @cbhua wants to fix some edge case that causes a bug, but it looks like the problem may not be the shared baseline itself

Reply:
@cbhua any news on this PR? Since the above seems to be correct, we may close this PR